MANIFEST.in

```diff
@@ -3,6 +3,7 @@ include LICENSE
 include MANIFEST.in
 include README.md
 include .pylintrc
+include .editorconfig
```

Comment:
I don't think this would be needed in a production source distribution.

Reply:
As I’m not a Python dev myself, where should I add it instead?

Reply:
I've now excluded it in the manifest